threedi_modelchecker/interfaces/raster_interface_rasterio.py

```diff
@@ -77,3 +77,7 @@ def min_max(self):
     @property
     def shape(self):
         return (self._dataset.height, self._dataset.width)
+
+    @property
+    def compression(self) -> Optional[str]:
```

Comment:
Return type doesn't match property in parent class

Reply:
thanks, fixed it